src/ParseError.js

```diff
@@ -15,8 +16,8 @@ class ParseError {
         // Error position based on passed-in Token or ParseNode.
 
     constructor(
-        message: string,              // The error message
-        token?: Token | ParseNode<*>, // An object providing position information
+        message: string,               // The error message
+        token?: ?Token | AnyParseNode, // An object providing position information
```

Comment:
Is `?` in front of `Token` necessary?

Reply:
Yes, so that a nullable value can be explicitly provided.